cards/styles2-cards.css

```diff
@@ -0,0 +1,445 @@
+/* Cards Main */
+.cards-container {
+  margin: 3rem;
+}
+
+.cards-main-heading {
+  color: yellow;
+}
+.cards-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
+  height: 200px;
+  border: 0;
+  transform: scale(1);
```

Comment:
you should use scale more then one because if we will use 1 then it will not show transform feature.

Reply:
actually, it was working and was getting too much of it that's why I have used 1